src/main/target/SPRACINGH7EXTREME/target.h

```diff
@@ -58,14 +58,6 @@
 #define QUADSPI1_MODE QUADSPI_MODE_BK1_ONLY
 #define QUADSPI1_CS_FLAGS (QUADSPI_BK1_CS_HARDWARE | QUADSPI_BK2_CS_NONE | QUADSPI_CS_MODE_LINKED)
 
-#define USE_FLASH_CHIP
```

Comment:
Target changes should go into separate pull requests.

Reply:
OK. Will split these out.

Reply:
The only change I think I need to make to these it to make the default blackbox logging to onboard FLASH as by default there's no SD card inserted. To be honest, that's just a personal preference. Do you think we should have an auto setting so the SD card is used if present, but otherwise the onboard FLASH is used?